README.md

```diff
@@ -70,16 +74,41 @@ As the Locker Service adjusts, I'll revisit this section to add resources that m
 
 - [D3](https://d3js.org/), a data visualization library.
 
-  - [C3](https://c3js.org/), a charting library dependent on D3
+- [C3](https://c3js.org/), a charting library dependent on D3
 
 - [ParaParse](https://github.com/mholt/PapaParse), a CSV parser, handles sloppy input and big files. (Use this)
 
 - [Leaflet](https://leafletjs.com/)
 
 - [Chroma.js](https://github.com/gka/chroma.js/)
 
-- [Signature Pad](https://github.com/szimek/signature_pad)
+- [Babylon.js](https://doc.babylonjs.com/)
```

Comment:
All of these do not seem relevant to LWC. These are general JS libraries.

Reply:
Also I wouldn't think some of these are locker safe... React.js for example would directly manipulate the DOM.